### PR TITLE
Feature/NP-1468 Cookie httpOnly property patch

### PR DESCRIPTION
**Pull request by celonicolosi**
Transfering this patch from PW to CA_testing, one of many PRs of NP-1468.
This patch fixes an issue with Selenium4 not bubbling up the httpOnly property of a cookie.

### CODE REVIEW

File: lib/ca_testing/patches/selenium_manager.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          See: https://github.com/SeleniumHQ/selenium/pull/8958 for more details/discussion including this fix
+        DESCRIPTION
+      end
+
+      def perform
+        ::Selenium::WebDriver::Manager.prepend CookieConverter
+      end
+
+      def deprecate?
+        false
```
Comment (luke-hill) [inline lib/ca_testing/patches/selenium_manager.rb:20]: Can we set the deprecated/prevented times to have the same date/times as the other selenium fix.

Comment (celonicolosi) [reply lib/ca_testing/patches/selenium_manager.rb:20]: Thanks, I wasn't sure about the deprecation date on this one. I'm going to change it to be the same as the other selenium fix.

File: lib/ca_testing/patches/selenium_manager.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          See: https://github.com/SeleniumHQ/selenium/pull/8958 for more details/discussion including this fix
+        DESCRIPTION
+      end
+
+      def perform
+        ::Selenium::WebDriver::Manager.prepend CookieConverter
+      end
+
+      def deprecation_notice_date
+        Time.new(2021, 12, 25)
```
Comment (luke-hill) [inline lib/ca_testing/patches/selenium_manager.rb:20]: Again. This is fixed in the currently selenium main. But won't be accessible to us until we migrate to selenium4 proper.

**Review by luke-hill — DISMISSED** [review]

**Review by mrdaniellewis — APPROVED** [review]
Looks good

**Review by luke-hill — APPROVED** [review]

**Pull request merged**